src/components/ContextData/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React,{ createContext, useContext, useState } from 'react'
+import data from '../../data'
+
+export const ContextData = createContext()
+
+export const DataProvider = ({ initalState = {}, children }) => {
+  const [values, setValues] = useState(initalState)
+
+  return (
+    <ContextData.Provider value={{ ...data, values, setValues }}>
+      {children}
+    </ContextData.Provider>
+  );
```

Comment:
This makes data (config and hex grid ), the values state object, and a setter for the values state object available to the entire app.